code/lib/types/src/modules/csf.ts

```diff
@@ -1,7 +1,111 @@
-import type { ViewMode as ViewModeBase } from '@storybook/csf';
+import type {
+  ViewMode as ViewModeBase,
+  AnnotatedStoryFn,
+  Args,
+  ArgsEnhancer,
+  ArgsFromMeta,
+  ArgsStoryFn,
+  ArgTypes,
+  ArgTypesEnhancer,
+  BaseAnnotations,
+  ComponentAnnotations,
+  ComponentId,
+  ComponentTitle,
+  Conditional,
+  DecoratorApplicator,
+  DecoratorFunction,
+  Globals,
+  GlobalTypes,
+  IncludeExcludeOptions,
+  InputType,
+  LegacyAnnotatedStoryFn,
+  LegacyStoryAnnotationsOrFn,
+  LegacyStoryFn,
+  LoaderFunction,
+  Parameters,
+  PartialStoryFn,
+  PlayFunction,
+  PlayFunctionContext,
+  Renderer,
+  SBArrayType,
+  SBEnumType,
+  SBIntersectionType,
+  SBObjectType,
+  SBOtherType,
+  SBScalarType,
+  SBType,
+  SBUnionType,
+  SeparatorOptions,
+  StepFunction,
+  StepLabel,
+  StepRunner,
+  StoryAnnotations,
+  StoryAnnotationsOrFn,
+  StoryContext,
+  StoryContextForEnhancers,
+  StoryContextForLoaders,
+  StoryContextUpdate,
+  StoryFn,
+  StoryId,
+  StoryIdentifier,
+  StoryKind,
+  StoryName,
+  StrictArgs,
+  StrictArgTypes,
+  StrictGlobalTypes,
+  StrictInputType,
+  Tag,
+} from '@storybook/csf';
```

Comment:
None of this should be imported. The export should be done differently (see next comments).
```suggestion
import type { ViewMode as ViewModeBase } from '@storybook/csf';
```